test/functional/test_compute_iterative_tv_map.py

```diff
@@ -66,3 +66,6 @@ def test_script_produces_consistent_results(
     )
     rho = np.corrcoef(result_amplitudes.to_numpy(), reference_amplitudes.to_numpy())[0, 1]
     assert rho > 0.95
+
+    # 4. regression, make sure no NaNs creep into metadata
+    assert not np.any(np.isnan(iterative_tv_metadata))
```

Comment:
This is slightly different in that it will also fail for infs, but I would usually write this assert as:

```
assert np.all(np.isfinite(iterative_tv_metadata))
```

Reply:
ah good suggestion! next time hit me before merge -- I'll do another mini-PR...